### PR TITLE
Use FPWD template for spec

### PR DESCRIPTION
**Pull request by rayankans**
issue #50

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -2,7 +2,8 @@
 Title: Contact Picker API
 Shortname: contact-picker
 Level: 1
-Status: UD
+Status: FPWD
+TR: https://w3c.github.io/contact-api/spec
```
Comment (rayankans) [inline spec/index.bs:6]: I didn't find a draft of the spec under https://www.w3.org/TR/, so I linked to the github spec for now

Comment (siusin) [reply spec/index.bs:6]: thanks a lot, @rayankans !

The "TR" meta-data is usually decided by our Sys-Team, so please hold this Pull Request until we publish the FPWD next Tuesday :)

Comment (siusin) [reply spec/index.bs:6]: Oh, sorry, the publication is waiting on a Former Objection to the new charter...

See [FPWD Request for Contacts Picker API #457](https://github.com/w3c/transitions/issues/457)

**Comment by marcoscaceres** [comment]
Closing as spec was published as FPWD.

**Pull request closed without merging**